short_tests/eddy_neknek/SIZE

```diff
@@ -0,0 +1,40 @@
+c
+c     Include file to dimension static arrays
+c     and to set some hardwired run-time parameters
+c
+      integer ldim,lx1,lxd,lx2,lx1m,lelg,lp,lelt,ldimt
+      integer ax1,ax2,lpx1,lpx2,lpelt,lbx1,lbx2,lbelt
+      integer toteq
+      integer lcvx1,lcvelt
+      integer lelx,lely,lelz,mxprev,lgmres,lorder,lhis
+      integer maxobj,maxmbr,lpert,nsessmax,nmaxl,nfldmax,nmaxcom
+
+      ! GENERAL
+      parameter (ldim=2)        ! domain dimension
+      parameter (lx1=8)         ! polynomial order; in 2D set lz1=1
+      parameter (lxd=12)        ! polynomial order for over-integration 
+      parameter (lx2=lx1-2)     ! polynomial order for pressure
+      parameter (lx1m=1)        ! polynomial order mesh solver; =1 no mesh motion
+
+      parameter (lelg=1000)     ! max total number of elements
+      parameter (lp=4)       ! max number of MPI ranks
+      parameter (lelt=250)      ! max number of elements per MPI rank 
+      parameter (ldimt=1)       ! max number of auxiliary fields (temperature + scalars)
+
+      ! OPTIONAL
+      parameter (lelx=1,lely=1,lelz=1)          ! global tensor mesh dimensions
+      parameter (ax1=1,ax2=1)                   ! averages
+      parameter (lbx1=1,lbx2=1,lbelt=1)         ! mhd
+      parameter (lpx1=1,lpx2=1,lpelt=1,lpert=1) ! linear stability
+      parameter (toteq=1)                       ! cmt
+      parameter (lcvx1=1,lcvelt=1)              ! cvode
+
+      parameter (mxprev=20,lgmres=30)           ! projection + Krylov space dimension
+      parameter (lorder=3)                      ! upper limit for order in time
+      parameter (lhis=1000)                     ! max history points per MPI rank
+      parameter (maxobj=4,maxmbr=lelt*6)        ! max number number of objects
+      parameter (nsessmax=2)                    ! multimesh parameters
+      parameter (nfldmax=4)                     ! multimesh parameters
```

Comment:
SIZE.inc also sets these variables in a safe way (rather than leaving it up to the user).   

``` fortran
parameter
     $  (nmaxl=min(1+(nsessmax-1)*2*ldim*lxz*lelt,2*ldim*lxz*lelt))
parameter 
     $  (nfldmax=min(1+(nsessmax-1)*(4+ldimt),4+ldimt))
```